src/components/Species/ImportSpeciesModal.tsx

```diff
@@ -33,12 +28,12 @@ export default function ImportSpeciesModal(props: ImportSpeciesModalProps): JSX.
       onClose={onClose}
       open={open}
       title={strings.IMPORT_SPECIES}
-      resolveApi={resolveSpeciesUpload}
+      resolveApi={SpeciesService.resolveSpeciesUpload}
       uploaderTitle={strings.IMPORT_SPECIES_LIST}
       uploaderDescription={strings.IMPORT_SPECIES_LIST_DESC}
-      uploadApi={uploadSpeciesFile}
-      templateApi={downloadSpeciesTemplate}
-      statusApi={getUploadStatus}
+      uploadApi={SpeciesService.uploadSpecies}
+      templateApi={SpeciesService.downloadSpeciesTemplate}
+      statusApi={SpeciesService.getSpeciesUploadStatus}
```

Comment:
in the import modal component, i added if/else handling for both types of responses.. now that species is also handled.. we can cleanup the if/else handling